web/src/content/posts/experiments-rails-like-form-helpers.mdx

```diff
@@ -0,0 +1,249 @@
+import { Tweet } from 'react-tweet'
+
+After ReactConf 2024 I was inspired by this tweet:
+
+<Tweet id="1791531154212033004" />
```

Comment:
@ahaywood The twitter card appears to maybe be taking on additional styling from being inside the `blog-post` class. Are you able to take a look at this?